Source/WebKit/GPUProcess/graphics/WebGPU/WebGPUObjectHeap.cpp

```diff
@@ -103,170 +103,142 @@ ObjectHeap::~ObjectHeap() = default;
 
 void ObjectHeap::addObject(WebGPUIdentifier identifier, RemoteAdapter& adapter)
 {
-    auto result = m_objects.add(identifier, Object { IPC::ScopedActiveMessageReceiveQueue<RemoteAdapter> { Ref { adapter } } });
-    ASSERT_UNUSED(result, result.isNewEntry);
+    m_objects.add(identifier, Object { IPC::ScopedActiveMessageReceiveQueue<RemoteAdapter> { Ref { adapter } } });
 }
 
 void ObjectHeap::addObject(WebGPUIdentifier identifier, RemoteBindGroup& bindGroup)
 {
-    auto result = m_objects.add(identifier, Object { IPC::ScopedActiveMessageReceiveQueue<RemoteBindGroup> { Ref { bindGroup } } });
-    ASSERT_UNUSED(result, result.isNewEntry);
+    m_objects.add(identifier, Object { IPC::ScopedActiveMessageReceiveQueue<RemoteBindGroup> { Ref { bindGroup } } });
 }
 
 void ObjectHeap::addObject(WebGPUIdentifier identifier, RemoteBindGroupLayout& bindGroupLayout)
```

Comment:
I think we should change the return type to bool, something like:
```
bool ObjectHeap::addObject(

...
return m_objects.add(identifier, Object { IPC::ScopedActiveMessageReceiveQueue<RemoteBindGroupLayout> { Ref { bindGroupLayout } } }).isNewEntry;
```

and then in the corresponding `Remote*` files, add a `MESSAGE_CHECK(isNewEntry);` so we get some indication of programmer error if this occurs